api/src/main/java/org/openmrs/module/clientregistry/api/model/CRPatient.java

```diff
@@ -0,0 +1,35 @@
+package org.openmrs.module.clientregistry.api.model;
+
+import ca.uhn.fhir.model.api.annotation.SearchParamDefinition;
+import ca.uhn.fhir.rest.gclient.TokenClientParam;
+import ca.uhn.fhir.rest.gclient.UriClientParam;
+import org.hl7.fhir.r4.model.Patient;
+
+public class CRPatient extends Patient {
```

Comment:
unclear if we need the CRPatient class or if these constants can go somewhere else and we deal with the Patient class directly

Reply:
I think we might. I am getting conflicts with the OpenMRS FHIR2 module using the Patient resource. Let's discuss during our call.